gen/spec.go

```diff
@@ -128,6 +129,19 @@ type TransformPass func(Elem) Elem
 // IgnoreTypename is a pass that just ignores
 // types of a given name.
 func IgnoreTypename(name string) TransformPass {
+	if strings.HasPrefix(name, "regex:") {
```

Comment:
```suggestion
	if name, ok := strings.CutPrefix(name, "regex:"); ok {
```

Reply:
Ah yeah. Knew there was one for it, but managed to overlook it :)